pkg/printers/internalversion/describe.go

```diff
@@ -1709,7 +1709,11 @@ func describeSecret(secret *api.Secret) (string, error) {
 		skipAnnotations := sets.NewString(api.LastAppliedConfigAnnotation)
 		printAnnotationsMultilineWithFilter(w, "Annotations", secret.Annotations, skipAnnotations)
 
-		w.Write(LEVEL_0, "\nType:\t%s\n", secret.Type)
+		if secret.Type == "" {
```

Comment:
our defaulting sets this to opaque.  How did you get an empty?  If you can't get an empty via deserialization, I'd rather tighten our validator.

@liggitt I'm assuming the defaulting and validation was you?